kuma/users/views.py

```diff
@@ -584,15 +592,16 @@ def form_valid(self, form):
     def get_context_data(self, **kwargs):
         context = super(SignupView, self).get_context_data(**kwargs)
 
-        # For GitHub users, find matching legacy Persona social accounts
-        assert self.sociallogin.account.provider == 'github'
+        # For GitHub/Google users, find matching legacy Persona social accounts
+        assert self.sociallogin.account.provider in ('github', 'google')
         or_query = []
         for email_address in self.email_addresses.values():
             if email_address['verified']:
                 or_query.append(Q(uid=email_address['email']))
         if or_query:
             reduced_or_query = reduce(operator.or_, or_query)
             matching_accounts = (SocialAccount.objects
+                                              .filter(provider='persona')
```

Comment:
You explained this to me on Zoom. Can you also write this down in a code comment so we can remember the why. (The why is that odd thing about UID being a int and sometimes an email)